backend/src/redis_pubsub.py

```
@@ -0,0 +1,84 @@
+    
+    def __init__(self):
+        self.redis_client = redis.from_url(config.redis_url)
+        self.pubsub = self.redis_client.pubsub()
+        self.channel = "job_updates"
+        self._running = False
+    
+    async def start(self, websocket_manager):
+        """Start listening for Redis messages."""
+        self._running = True
+        self.pubsub.subscribe(self.channel)
+        
```

Ruff (0.12.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)